test/cases/no dollars in identifiers.c

```diff
@@ -0,0 +1,7 @@
+void foo$() { }
+
+void fib() {
+  int $test;
+}
+
+void ano$ther() {}
```

Comment:
I didn't added the ``no dollars in identifiers`` case because I think sooner or later, we would need more such options and thus a proper solution is needed, like parsing the whole (or atleast a larger) set of cli args from the first comment?